README.md

```diff
@@ -8,16 +8,21 @@ _Example screenshot of flask_weather application_
 
 ## Setup
 
-- Install dependencies
+- Install dependencies (for the React application you will need NodeJS and yarn)
 - `pip install -r requirements.txt`
+- `yarn install`
 - Add environment variables
   - `weather_key="###############";`. The weather_key is the API key received from registering at https://darksky.net/dev.
   - add `secret_key="********"`. It should be a random string that is hard to guess.
+- Build React app
+  - `yarn build`
```

Comment:
Can this be combined to `yarn build && python app.py`?

Reply:
> Can this be combined to `yarn build && python app.py`?

I did not put them both in the same command since yarn build needs to be run just once till any change is made to the React application.